.circleci/config.yml

```diff
@@ -1591,6 +1591,30 @@ workflows:
             - attach_workspace:
                 at: /
 
+  GCP-Daily-Services-Query-Restart-Performance-6N-6C:
```

Comment:
Please also modify `GCP-Weekly-Services-NetDelay-15N-1C:` results folder to `results/15N_1C/NetDelay`

Reply:
Changed already.